ggshield/iac/models/iac_policy_break.py

```diff
@@ -0,0 +1,42 @@
+from typing import Any, Dict, List
+
+from marshmallow import fields, post_load
+from pygitguardian.models import PolicyBreak, PolicyBreakSchema
+
+from ggshield.iac.models.iac_match import IaCMatch, IaCMatchSchema
+
+
+class IaCPolicyBreakSchema(PolicyBreakSchema):
+    matches = fields.List(fields.Nested(IaCMatchSchema), required=True)
+    policy_details = fields.Dict(
+        fields.String(required=True), fields.String(required=True)
+    )
+
+    # Override parent method to prevent using the parent deserializer
+    def make_policy_break(self, data: Dict[str, Any], **kwargs: Any) -> Dict[str, Any]:
+        return data
+
+    @post_load
+    def make_iac_policy_break(
+        self, data: Dict[str, Any], **kwargs: Any
+    ) -> "IaCPolicyBreak":
+        return IaCPolicyBreak(**data)
+
+
+class IaCPolicyBreak(PolicyBreak):
+    SCHEMA = IaCPolicyBreakSchema()
+
+    def __init__(
```

Comment:
We will need the `sha` of the policy break as well.

Reply:
Should we have it as such or can it be added in the policy_details?

Reply:
Well the sha would be computed both from policy and matches, so it does not really belong in policy_details IMO.

Reply:
Ha, yes, didn't understand you were talking about the identifier. I called it policy_break_id for now, we can change the name after your work on determining a vulnerability ID if there is a better match